src/webservices.js

```diff
@@ -4,31 +4,37 @@ import config from "./config"
 
 
 function fetchJson(url, options) {
-  return fetch(url, options).then(parseJsonResponse)
+  return fetch(url, options)
+    .then(response => response.json()
+      .then(data => {
+        if (response.status >= 200 && response.status < 300) {
+          return {
+            data,
+            'country-package': response.headers.get('country-package'),
+            'country-package-version': response.headers.get('country-package-version'),
+          }
+        } else {
+          throw new Error(JSON.stringify(data.error))
```

Comment:
`data.error` might be empty if `response.status` is a 3XX. Need to handle that case.